### PR TITLE
Belindas-Closet-nextjs-8-contact-page-unit-test

### LINKED ISSUE #450: Unit Test: Contact Page unit Test
- [x] This issue is made to resolve the lack of unit test for the contact page

### PR DESCRIPTION
**Pull request by MaddasaHatter**
Resolves #450 
This PR resolves the lack of unit tests for the contact page
Screenshot 2024-06-10 213153

### CODE REVIEW

**Review by Seiyaroo — APPROVED** [review]
With the images provided going to say it looks good since its passing. LGTM?

**Review by Robel-003 — APPROVED** [review]
Working, and test is passing. Great job! One thing I'd suggest to specify a path to only run the contact page test, otherwise you'd have to run all your test every time. 
To only run the contact page test file, you can specify the path ("test:contact": "jest tests/unit/contact-page/contactpage.test.tsx") under scripts in the package.json  

Screenshot 2024-06-11 at 8 53 22 PM

**Review by ogarmela — APPROVED** [review]
It looks good and it's working.

**Review by intisarosman1 — APPROVED** [review]
The tests pass successfully. LGTM
Screenshot 2024-06-12 010038

**Review by ParysUnangst — COMMENTED** [review]
Looks great and passing!

File: tests/unit/contact-page/contactpage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (ParysUnangst) [inline tests/unit/contact-page/contactpage.test.tsx:1]: :)

File: tests/unit/contact-page/contactpage.test.tsx
Comment (ParysUnangst) [inline tests/unit/contact-page/contactpage.test.tsx:1]: Looks great and passing!

**Pull request merged**